react/features/notifications/components/AbstractNotificationsContainer.js

```diff
@@ -177,7 +177,7 @@ export default class AbstractNotificationsContainer<P: Props>
  */
 export function _abstractMapStateToProps(state: Object) {
     const { notifications } = state['features/notifications'];
-    const _visible = areThereNotifications(state);
+    const _visible = areThereNotifications();
```

Comment:
This change is from you? Because about downward compatibility, i would suggest not doing it. What do you think?

Reply:
This change is from me. I changed the `areThereNotifications()` to only return false due to removing all the notifications/warnings Jitsi has. In my perspective, I do not see any downward compatibility on this change. However, I am okay to get back the original changes. Please say so.

Reply:
You mean the pop button left in the OVS will never triggert, now? Ok i am fine with this as quick fix 👍